src/benchmarks/micro/libraries/System.Drawing/Perf_Image_Load.cs

```diff
@@ -59,21 +59,23 @@ private static ImageTestData[] CreateTestCases()
                     new ImageTestData(ImageFormat.Gif)
                 };
             }
-            catch (Exception) when (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            catch (Exception) when (IsInLab && RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
             {
                 Console.ForegroundColor = ConsoleColor.Red;
                 Console.WriteLine("libgdiplus is missing, you can install it by running 'apt-get install libgdiplus'");
                 Console.ResetColor();
 
                 throw;
             }
-            catch (Exception) when (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+            catch (Exception) when (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                // skip tests on macOS instead of failing
+                // skip tests on Linux and macOS instead of failing
```

Comment:
Do you want to print a warning about the test being skipped?

Reply:
good point, I've changed the logic to always print warning but throw only for Lab runs